src/PointwiseFunctions/GeneralRelativity/Tags.hpp

```diff
@@ -143,6 +143,17 @@ template <size_t Dim, typename Frame, typename DataType>
 struct WeylElectric : db::SimpleTag {
   using type = tnsr::ii<DataType, Dim, Frame>;
 };
+
+/*!
+ * \brief Computes the scalar \f$E_{ij} E^{ij}\f$ from the electric part of the
+ * Weyl tensor \f$E_{ij}\f$ and the inverse spatial metric \f$\gamma^{ij}\f$,
+ * i.e. \f$E_{ij} = \gamma^{ik}\gamma^{jl}E_{ij}E_{kl}\f$.
```

Comment:
the indices don't match on each side of this equation; I don't know if you want `\f$E_{ij} E^{ij}\f$` or define a new symbol for the scalar earlier in this sentence